helm-chart/amalthea-sessions/values.yaml

```diff
@@ -36,3 +36,16 @@ sidecars:
   image:
     repository: renku/sidecars
     tag: latest
+csi-rclone:
+  install: false
+  # This section is only relevant if you are installing csi-rclone as part of Amalthea.
+  # When this chart is deployed as part of Renkulab the following value should match the name of the storage class
+  # configured for Notebooks cloudstorage.
+  ## Name of the csi storage class to use for RClone/Cloudstorage. Should be unique per cluster.
+  # storageClassName: csi-rclone
```

Comment:
I think that one should be `csi-rclone-secret-annotation` since it's the method used to for the defining the configuration.

Reply:
I think we should remove all the commented stuff. Just point people to https://github.com/SwissDataScienceCenter/csi-rclone/blob/master/deploy/csi-rclone/values.yaml so that they check what values are available. And be default the default values of csi-rclone should work.

Reply:
I removed the comments and updated the default value to match the current default value of the child helm chart.

Reply:
Thank you for spotting this and your comment @sgaist.

There was more to it as you might have seen from the several corrections I had to apply on top of my first patch.